LiteDB/Engine/Services/RebuildService.cs

```diff
@@ -22,14 +22,19 @@ public RebuildService(EngineSettings settings)
         {
             _settings = settings;
 
-            // open, read first 16kb, and close data file
-            var buffer = this.ReadFirstBytes();
+			// test for prior version
```

Comment:
please change the indentation to be space instead of tabs

Reply:
Done.